war/src/main/js/widgets/config/tabbar.less

```diff
@@ -151,156 +166,83 @@
         }
       }
 
-      .section-header{
-        margin: 30px 0px 5px -10px;
-        font-size: 1.25rem;
-        padding: 10px 15px;
-        border: none;
-        border-top: 1px solid #ddd;
-      }
-      .attach-previous {
-        margin-left:3px;
-      }
-      .validation-error-area td{
-        padding:0 5px;
+      // Help & descriptions
+      .help {
+        background: var(--configure-job-help-area-bg-color);
       }
 
-      .stapler-class-bag td {
-        padding: 0px;
-      }
-      
-      td.setting-name{
-        line-height:29px;
-      }
-      td.setting-main{
-        input[type='radio'],
-        input[type='checkbox']{
-          position:relative;
-          top:.33em;
-        }
+      p,
+      .help,
+      .setting-description {
+        font-size: var(--font-size-base);
       }
-      td{
-        padding:5px;
-        
-        textarea.setting-input {
-          display:block;
-        }
 
-        input[type='email'].setting-input,
-        input[type='password'].setting-input,
-        input[type='text'].setting-input{
-          height:var(--input-line-height);
-          line-height:var(--input-line-height);
-          padding:0 5px;
-          .border-radius(@border-radius);
-        }
+      tr.help-area,
+      .tr.help-area {
+        background: none !important;
 
-        select:not([multiple]){
-          height:var(--input-line-height);
-          line-height:var(--input-line-height);
-        }
-        select:not(.select-ajax-pending) {
-          border-radius:0;
-          background:var(--brightest-bg-color);
-        }
-        .help{
-          background:var(--configure-job-help-area-bg-color);
-        }
-        p,
-        .help,
-        .setting-description {
-          font-size: var(--font-size-base);
-        }
-      }
-      td[colspan="4"]:empty{
-        display:none
-      }
-      td.setting-help{
-        vertical-align:top;
-        padding:5px 10px 5px 5px
-      }
-      td.setting-leftspace:after {
-        content:' '; 
-        display:block; 
-        width:13px; 
-        height:1px;
-      }
-      .repeated-chunk,
-      tr[nameref^="rowSetStart"],
-      tr[nameref^="cb"],
-      tr[nameref^="radio-block"] {
-        background:var(--shade-hint); 
-        border-left:5px solid var(--shade)
```

Comment:
These rules should be back in, otherwise it's tough to properly appreciate the nesting:

<img width="981" alt="Captura de pantalla 2020-07-01 a las 9 19 09" src="https://user-images.githubusercontent.com/5738588/86215368-ca254380-bb7c-11ea-850e-81e46a4627fe.png">

Reply:
Re-applied:

![image](https://user-images.githubusercontent.com/21194782/86537641-8fa70800-bee8-11ea-9215-5285ac20de3c.png)